a11y additions

mxmason opened the pull request:
Extends from as https://github.com/Thinkful-Ed/redux-hot-cold/pull/5 - now with testing!

iameli committed:
remove references to fewestGuesses

This maybe used to be a feature, but isn't anymore? These two lines certainly aren't doing anything,
and they foul up tests that check to make sure we have the expected properties in the state after
the NEW_GAME action runs.

Commit:
Merge pull request #1 from iameli/remove-fewest-guesses

oampo committed:
Per-component stylesheets

oampo committed:
Update store.js

oampo committed:
Remove Thunk from package.json

oampo committed:
Merge pull request #3 from Thinkful-Ed/remove-thunk

Remove last vestiges of Thunk

oampo committed:
Update to React 16

Rosuav committed:
Floor random rather than rounding it

Using round() with Math.random() skews the output away from the extremes. The safe way to use Math.random() to generate integers is with floor.

Cherry-picked from react-hot-cold#46fb1450e74e9eb28d88326dbc4fcc6d9f9a48a3

oampo committed:
Merge pull request #4 from Rosuav/patch-1

Floor random rather than rounding it

mxmason committed:
Update deps to react16, etc

mxmason committed:
Add package lock

mxmason committed:
Reduxify based on a11y refactor fo app

mxmason committed:
Un-hide sr desciption, add space before it

mxmason committed:
Remove whitespace from auralstatus

mxmason committed:
Style cleanup & literal html entities

mxmason committed:
Rename variable, edit comment

mxmason committed:
Merge branch 'redux/feature/a11y-additions' into reduxtesting/feature/a11y-additions

mxmason committed:
Merge tests from updated react app

mxmason committed:
Delete modal css

mxmason committed:
version bump

mxmason committed:
Update reducer tests to reflect new app

mxmason committed:
Take state logic out of game test

mxmason committed:
Create test for Status Section

mxmason committed:
Remove duplicate css import

mxmason committed:
All tests passing

oampo commented on src/components/top-nav.test.js:
```diff
@@ -1,34 +1,20 @@
+    link.simulate('click', {
+      preventDefault() {}
     });
+    expect(dispatch).toHaveBeenCalled();
```
We've lost the check for the type of the action here

oampo commented on src/components/top-nav.test.js:
```diff
@@ -1,34 +1,20 @@
-        });
-        expect(dispatch).toHaveBeenCalled();
-        expect(dispatch.mock.calls[0][0]).toEqual(toggleInfoModal());
+  it('Should call onNewGame when new game is clicked', () => {
```
Name doesn't match with the test content

oampo commented on src/components/guess-form.test.js:
```diff
@@ -1,28 +1,28 @@
-        wrapper.simulate('submit');
-        expect(dispatch).toHaveBeenCalledWith(makeGuess(value));
-    });
+  it('Should fire the onMakeGuess callback when the form is submitted', () => {
```
Name doesn't match with the test content

oampo reviewed (changes requested):
Couple of small changes, then we're ready to roll!

mxmason committed:
Fixes per joe, add test for dispatching of generateAuralUpdate

mxmason committed:
Remove excess import

mxmason commented:
Addressed your comments! Also noticed I failed to test that `generateAuralUpdate` was dispatched, so I fixed that. Went back and added a test for the `onGenerateAuralUpdate` callback in https://github.com/Thinkful-Ed/react-hot-cold-testing/pull/2 as well

oampo committed:
Replace &apos

oampo committed:
Fix test name

oampo committed:
Remove document.title setting

Might tackle later, but this should be a component, and definitely not
the reducer

oampo committed:
Move correctAnswer to action

Makes the reducer deterministic

oampo committed:
Update tests for correctAnswer in action

oampo committed:
Add actions test
